src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationDetailViewController.swift

```diff
@@ -57,7 +57,7 @@ extension AppInformationDetailViewController {
 
 extension AppInformationDetailViewController: UITextViewDelegate {
 	func textView(_ textView: UITextView, shouldInteractWith url: URL, in characterRange: NSRange, interaction: UITextItemInteraction) -> Bool {
-		WebPageHelper.openSafari(withUrl: url, from: self)
+		UIApplication.shared.open(url, options: [:], completionHandler: nil)
```

Comment:
Technically this can be removed altogether, as iOS handles links and phone number automatically if no delegate is provided, if I remember correctly. Please double check.

Reply:
I would double on this too.

Reply:
could be removed!